modules/tls/src/test/java/org/mule/runtime/module/tls/DefaultTlsContextFactoryTestCase.java

```diff
@@ -34,6 +37,10 @@
 
   @BeforeClass
   public static void createTlsPropertiesFile() throws Exception {
+
+    // This test will be ignored when the Allure framework is used because of this issue MULE-10805
+    assumeThat(System.getProperty("allure.profile.is.activated", "false"), Is.is(equalTo("false")));
```

Comment:
If you use an assumption you will never be aware that this test is not running. The test will show as Passed while it didn't actually run.
We should use an @ Ignore and place a TODO tag.